testing/endtoend/types/fork.go

```diff
@@ -45,7 +56,26 @@ func InitForkCfg(start, end int, c *params.BeaconChainConfig) *params.BeaconChai
 	}
 	// Time TTD to line up roughly with the bellatrix fork epoch.
 	// E2E sets EL block production rate equal to SecondsPerETH1Block to keep the math simple.
-	ttd := uint64(c.BellatrixForkEpoch) * uint64(c.SlotsPerEpoch) * c.SecondsPerSlot
+	// If Bellatrix is disabled (math.MaxUint64), the chain starts post-merge so TTD should be 0.
+	ttd := uint64(0)
+	if start > version.Bellatrix {
```

Comment:
Something is wrong here. My suggestion was incorrect because the correct condition is `if start <= version.Bellatrix`. But if the Electra genesis test passes with this code, doesn't that mean that when starting after Bellatrix, the Bellatrix fork is `0` and not maxuint, and that's why there is no overflow? Which would mean the original code is OK